### ISSUE TITLE
Z buffering

### ISSUE THREAD

**Issue by carlo-theunissen** [carloboy7/UrbanRoad#36]
If you place a block on top of another you'll get some z buffering.

<img width="521" alt="afb" src="https://cloud.githubusercontent.com/assets/7584025/14351242/754de8a6-fcce-11e5-96cd-af33a0be0666.png">

**Comment by WouterStrangeDuck** [comment]
we could put the buildings 1 pixel up and the problem is fixed. easiest way is probably by code. Best way would be editing the pivot of the building but then we have to adjust alot...

**Comment by carlo-theunissen** [comment]
Yeah we are going to fix that in code, that is way simpler then moving the pivot point.
